### DISCUSSION TITLE
Shareable config not working despite DANGEROUS_EXTEND

### DISCUSSION THREAD

**Discussion by nibe** [browserslist/browserslist#777] [General]
I have set `BROWSERSLIST_DANGEROUS_EXTEND=1` on my system (`echo` works), yet I keep getting the error `Browserslist config needs 'browserslist-config-' prefix.  Use 'dangerousExtend' option to disable.`

package.json:
```
{
  "dependencies": {
    "bootstrap": "^5.3.0",
    "postcss": "^8.4.24",
    "postcss-cli": "^10.1.0",
    "autoprefixer": "^10.4.14"
  },
  "browserslist": [
    "extends bootstrap"
  ]
}
```

How to use `BROWSERSLIST_DANGEROUS_EXTEND` or `dangerousExtend` via Autoprefixer?

**Comment by ai** [comment]
> How to use BROWSERSLIST_DANGEROUS_EXTEND or dangerousExtend via Autoprefixer?

You can use shell env variables by calling build script something like:

```sh
BROWSERSLIST_DANGEROUS_EXTEND=1 npm build
```

> "extends bootstrap"

Sharable config is a special npm package which use special API to export the config for target browsers.

`bootstrap` npm package doesn’t use this feature (it has `.browserslistrc` file on Github repo, but it is not `index.js` which we require for sharable config.

Seems like you are doing a wrong thing.

**Reply by nibe** [reply]
OK thanks. So Browserslist does not support automatically using the Browserslist of another project, unless it is a special sharable config? In that case I will have to keep track of changes manually or copy the .browserslistrc file as part of the build.

**Reply by ai** [reply]
> Browserslist does not support automatically using the Browserslist of another project, unless it is a special sharable config?

Yes

> In that case I will have to keep track of changes manually or copy the .browserslistrc file as part of the build.

Yes. But also you can create that npm package if you want.
